app/src/main/java/com/jetbrains/handson/mpp/mobile/MainActivity.kt

```diff
@@ -28,7 +26,7 @@ class MainActivity : AppCompatActivity(), ApplicationContract.View {
     }
 
     override fun showResults(result: List<JourneyTableDataElem>) {
-        findViewById<TextView>(R.id.search_results).text = result.toString()
+        findViewById<RecyclerView>(R.id.search_results).adapter = SearchResultAdapter(result)//.text = result.toString()
```

Comment:
Again I think you can delete the `findViewById<RecyclerView>(R.id.`. Also can delete the comment at the end?